src/view/forms/organization/Organization.tsx

```diff
@@ -3,18 +3,41 @@ import {
   createOrgAction,
   ICreateOrgActionStrict,
   createOrgTransaction,
+  updateOrgAction,
+  updateOrgTransaction,
+  IContactStrict,
+  IFactoryAddressStrict,
 } from 'services/protobuf/organization';
-import { Organization } from 'services/protobuf/compiled';
+import {
+  createTransferAssertionAction,
+  createTransferAssertionActionTransaction,
+} from 'services/protobuf/assertion';
+import {
+  Organization,
+  IUpdateOrganizationAction,
+} from 'services/protobuf/compiled';
+import { isDataClaimed } from 'utils';
 import { VpnKey as Key } from '@material-ui/icons';
 import { Button, Typography, Grid, TextField } from '@material-ui/core';
 import { SelectOrganizationType } from 'view/forms/organization/SelectOrganizationType';
 import { createBatch } from 'services/protobuf/batch';
 import { useAuth } from 'services/hooks';
 import { FormErrMsg, TransactionFormProps } from 'view/forms/utils';
-import { postBatches } from 'services/api';
+import { postBatches, FactoryResData } from 'services/api';
+import { getSignerPubKeyHex } from 'services/crypto';
 import { CreateContactForm } from './CreateContact';
 import { CreateFactoryAddressForm } from './CreateFactoryAddress';
 
+export interface OrgTransactionFormProps extends TransactionFormProps {
+  existingOrg: FactoryResData;
+}
+
+export interface IUpdateOrganizationActionStrict
+  extends IUpdateOrganizationAction {
+  contacts: NonNullable<IContactStrict[]>;
```

Comment:
I believe both of these should be like this:

```suggestion
  contacts: NonNullable<IUpdateOrganizationAction['contacts]>;
```

Reply:
So those are different types I believe, one is strict and the other is not.